packages/ui-extensions-go-cli/project.json

```diff
@@ -6,6 +6,13 @@
     "implicitDependencies": ["ui-extensions-cli"],
     "tags": ["scope:feature", "scope:ui-extensions"],
     "targets": {
+      "build": {
+        "executor": "nx:run-script",
+        "dependsOn": ["^build"],
```

Comment:
Building the Go CLI requires building the Dev console. The `^build` value for `dependsOn` means "run build in all my direct and transitive dependencies".